README.md

```diff
@@ -173,34 +171,34 @@ The Equinix Metal authentication token can be stored in the `$PACKET_TOKEN` envi
 Environment variable:
 
 ```bash
-export PACKET_TOKEN=[PACKET_TOKEN]
+export METAL_AUTH_TOKEN=[METAL_AUTH_TOKEN]
 ```
 
-YAML configuration file - `$HOME/.packet-cli.yaml`:
+YAML configuration file - `$HOME/.config/equinix/.metal.yaml`:
```

Comment:
Within XDG, this doesn't have to be _hidden_, this should be `metal.yaml` (no dot).